secure_message/services/internal_user_service.py

```diff
@@ -1,12 +1,44 @@
 import logging
 
+from flask import current_app
+import requests
+from requests import HTTPError
 from structlog import wrap_logger
 
 logger = wrap_logger(logging.getLogger(__name__))
 
 
 class InternalUserService:
     @staticmethod
-    def get_user_details(uuid):  # NOQA pylint:disable=unused-argument
+    def get_user_details(uuid):
         """gets the user details from the internal user service"""
-        logger.debug("getting user details from uaa")
+        logger.info("Getting user details from uaa", uuid=uuid)
+        url = f"{current_app.config['UAA_URL']}/Users/{uuid}"
+        uaa_token = current_app.oauth_client_token
+        headers = {'Accept': 'application/json',
+                   'Authorization': 'Bearer ' + uaa_token.get('access_token'),
+                   'Content-Type': 'application/json'}
+
+        try:
+            response = requests.get(url, headers=headers)
+            response.raise_for_status()
+            resp_json = response.json()
+        except HTTPError:
+            logger.exception("Failed to get user info", uuid=uuid)
+            raise
+        except ValueError:
+            logger.exception("Failed to decode response JSON.", uuid=uuid)
```

Comment:
:art: Totally minor and ignore if you wish :) Extra full stop on the end of the message where other messages don't have one.